UnitTests/ResourceManager.Tests/CommitDataRenders/CommitDataHeaderRendererIntegrationTests.cs

```diff
@@ -49,12 +49,12 @@ public void Setup()
         [Test]
         public void Render_with_tabs_and_links()
         {
-            var expectedHeader = "Author:			<a href='mailto:John.Doe@test.com'>John Doe (Acme Inc) &lt;John.Doe@test.com&gt;</a>" + Environment.NewLine +
-                                 "Author date:	3 days ago (" + LocalizationHelpers.GetFullDateString(_data.AuthorDate) + ")" + Environment.NewLine +
-                                 "Committer:		<a href='mailto:Jane.Doe@test.com'>Jane Doe &lt;Jane.Doe@test.com&gt;</a>" + Environment.NewLine +
+            var expectedHeader = "Author:		<a href='mailto:John.Doe@test.com'>John Doe (Acme Inc) &lt;John.Doe@test.com&gt;</a>" + Environment.NewLine +
+                                    "Author date:	3 days ago (" + LocalizationHelpers.GetFullDateString(_data.AuthorDate) + ")" + Environment.NewLine +
```

Comment:
What is wrong here? Should I fix something? It is merged already.

Reply:
Indentation is not matching
No need to adjust this now, it will be noise when changed at a future follow up
Such whitespace changes could be ignored by blame though: https://www.moxio.com/blog/43/ignoring-bulk-change-commits-with-git-blame